er/data/fetchData.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+readonly DATA_URL='https://linqs-data.soe.ucsc.edu/public/psl-examples-data/er/er_tiny.tar.gz'
```

Comment:
Good tactics with the move.
Switch these around some so you only specify 'er_tiny' once.

Also, rename `DATA_MOVE` to something more descriptive like `RAW_DATA_NAME`.